cfme/storage/volume.py

```diff
@@ -227,24 +243,56 @@ def refresh(self):
         self.provider.refresh_provider_relationships()
         self.browser.refresh()
 
+    def create_backup(self, name, incremental=None, force=None):
+        """create backup of cloud volume"""
+        view = navigate_to(self, 'Backup')
+        view.backup_name.fill(name)
+        view.incremental.fill(incremental)
+        view.force.fill(force)
+
+        view.save.click()
+        view.flash.assert_success_message('Backup for Cloud Volume "{}" created'.format(self.name))
+
+        wait_for(lambda: self.backups > 0, delay=20, timeout=1000, fail_func=self.refresh)
+
     @property
     def exists(self):
         try:
             navigate_to(self, 'Details')
             return True
-        except VolumeNotFound:
+        except VolumeNotFoundError:
             return False
 
     @property
     def size(self):
+        """ size of storage cloud volume.
+
+        Returns:
+            :py:class:`int' size of volume.
+        """
         view = navigate_to(self, 'Details')
         return view.entities.properties.get_text_of('Size')
```

Comment:
REQUIRED: you mention int class but still return text

Reply:
my mistake; I don't want touch volume size property. Maybe due to this other test fails. changing doc string to string. actually, it returning '1 GB'